env.default

```diff
@@ -15,7 +15,7 @@ ASSET_DOMAIN=network.mofoprod.net
 CONTENT_TYPE_NO_SNIFF=True
 CORS_WHITELIST=*
 CORS_REGEX_WHITELIST=
-DATABASE_URL=
+DATABASE_URL=postgres://postgres@postgres:5432/postgres
```

Comment:
I tried using environment variables in the docker-compose.yml file but had no luck there.

Reply:
What I mean is that in docker-compose I tried setting `POSTGRES_PASSWORD`, `POSTGRES_USER`, and `POSTGRES_DB` to override the default user/pass/db combo on container creation but it never seemed to work right.